src/main/fox_hud.c

```diff
@@ -52,19 +119,56 @@ extern u8 D_5004580[];
 extern u8 D_5004DC0[];
 extern u8 D_5004E20[];
 extern u8 D_5005460[];
+extern u8 D_5009F60[];
+extern u8 D_500A050[];
+extern u8 D_500A140[];
+extern u8 D_500A230[];
+extern u8 D_500A320[];
+extern u8 D_500A410[];
+extern u8 D_500A500[];
+extern u8 D_500A5F0[];
+extern u8 D_500A6E0[];
+extern u8 D_500A7D0[];
 extern u16 D_6000840[];
 extern u16 D_6000C80[];
+extern u8 D_6000D80[];
 extern u8 D_6001260[];
 extern u8 D_60012D0[];
 extern u8 D_6002890[];
 extern Gfx D_6003090[];
 
-void func_80084930(f32, f32, s32);
-void func_80086444(void);
-void func_80087788(void);
-void func_801BE034(Actor*, Player*);
+// DATA SECTION START
+
+s16 D_800D1970 = 0;
+
+// unused
+s16 D_800D1974[] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+
+f32 D_800D19AC[] = { 255.0f, 255.0f, 255.0f, 255.0f, 255.0f, 255.0f, 255.0f, 255.0f, 255.0f, 0.0f, 0.0f, 0.0f, 0.0f };
```

Comment:
Judging by its use below, this is f32[5]. The stuff after it then makes most sense as two f32[4].